Add support for WebP format to Gd2 image adapter. This allows for automatic resizing of webp images.
You can check if the GD plugin supports webp by calling `php -r "print_r(gd_info());"`
======================================================================================
--- a/Image/Adapter/Gd2.php
+++ b/Image/Adapter/Gd2.php	(date 1733475399466)
@@ -32,6 +32,7 @@
         IMAGETYPE_PNG => ['output' => 'imagepng', 'create' => 'imagecreatefrompng'],
         IMAGETYPE_XBM => ['output' => 'imagexbm', 'create' => 'imagecreatefromxbm'],
         IMAGETYPE_WBMP => ['output' => 'imagewbmp', 'create' => 'imagecreatefromxbm'],
+        IMAGETYPE_WEBP => ['output' => 'imagewebp', 'create' => 'imagecreatefromwebp'],
     ];
 
     /**
@@ -207,6 +208,10 @@
             case IMAGETYPE_JPEG:
                 $quality = $this->quality();
                 break;
+
+            case IMAGETYPE_WEBP:
+                $quality = $this->quality();
+                break;
 
             default:
                 $quality = null;    // No compression.
@@ -404,6 +409,9 @@
         }
         if (IMAGETYPE_JPEG === $fileType) {
             $isTrueColor = true;
+        }
+        if (IMAGETYPE_WEBP === $fileType) {
+            $isTrueColor = true;
         }
         return false;
     }
